websites/A/Airbnb/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "993151156652093480",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		},
+		privacy = await presence.getSetting("privacy");
+
```

Comment:
`document.location.pathname` is repeated many times so you should deconstruct it and reference `pathname instead`

`{ pathname } = document.location`